src/components/SocialMediaLinks/SocialMediaLinks.stories.tsx

```diff
@@ -0,0 +1,261 @@
+import Heading from "../Heading/Heading";
+import { Source } from "@storybook/blocks";
+
+const meta: Meta<typeof SocialMediaLinks> = {
+  title: "Components/Navigation/SocialMediaLinks",
+  component: SocialMediaLinks,
+  decorators: [withDesign],
+  argTypes: {
+    borders: {
+      table: { disable: true },
+    },
+    className: { table: { disable: true } },
+    color: {
+      table: { disable: true },
+    },
+    id: { table: { disable: true } },
+    layout: {
+      table: { disable: true },
+    },
+    linksData: { table: { disable: true } },
+    showLabels: { table: { disable: true } },
+    size: {
+      table: { disable: true },
+    },
+  },
+  parameters: {
+    controls: {
+      hideNoControlsWarning: true,
+    },
+    design: {
+      type: "figma",
+      url: "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=77103-126004&t=Yx1GyAfBJyhuGIGs-4",
+    },
+    jest: ["SocialMediaLinks.test.tsx"],
+  },
+};
+export default meta;
+type Story = StoryObj<typeof SocialMediaLinks>;
+
+export const WithControls: Story = {
+  args: {
+    borders: "none",
+    color: "textDefault",
+    className: undefined,
+    id: undefined,
+    layout: "row",
+    linksData: [
+      { type: "facebook" },
+      {
+        type: "twitter",
+        url: "twitter.com/elsewhere",
+        labelText: "Alt Twitter",
+      },
+      {
+        type: "instagram",
+      },
+    ],
+    showLabels: true,
+    size: "small",
+  },
+  argTypes: {
+    borders: {
+      control: { type: "radio" },
+      options: borderTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "none" },
+      },
+    },
+    className: {
+      control: true,
+      table: {
+        disable: false,
+      },
+    },
+    color: {
+      description:
+        "Any of three optional values that will change the color of the svg and label text (if any).",
+      control: { type: "radio" },
+      // @todo the table will not display these options.
+      options: colorTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "textDefault" },
+      },
+    },
+    id: {
+      control: true,
+      table: {
+        disable: false,
+      },
+    },
+    layout: {
+      control: { type: "radio" },
+      options: layoutTypesArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "row" },
+      },
+    },
+    linksData: { table: { disable: false } },
+    showLabels: {
+      table: {
+        disable: false,
+        defaultValue: { summary: false },
+      },
+    },
+    size: {
+      control: { type: "radio" },
+      options: sizeTypeArray,
+      table: {
+        disable: false,
+        defaultValue: { summary: "small" },
+      },
+    },
+  },
+  parameters: {
+    // This is gray b/c textInverse would be invisible otherwise.
+    // @todo It is possible to make the background change based on controls. But this requires changes to preview.tsx
```

Comment:
There's a separate control in the _top nav_ that allows you to change the background color. That should be used instead of updating through the Controls.

<img width="521" alt="Screen Shot 2023-09-07 at 3 44 37 PM" src="https://github.com/NYPL/nypl-design-system/assets/1280564/c64fa563-6fae-479a-a3fe-f388957c8462">

So this should be the default to be consistent with other components.

Reply:
It seems weird to me that if I select "textInverse" in the controls the icons and text become invisible. As a newbie I did not realize -- until later -- that I could control the background color via the toolbar. \*whine\* And then I have to do _another_ click! \*whine\*

Admittedly as implemented this is problematic b/c the background doesn't change unless you click any of the controls, making it seem like that change is part of that control.

Personally I would prefer to see the background change automatically for when `textInverse` is true.

@bigfishdesign13 What do you think?

Reply:
A precedent has already been set to update the bg color when needed.  Please see the docs for the `Logo` component.

https://nypl.github.io/nypl-design-system/reservoir/v1/?path=/story/components-media-icons-logo--with-controls&args=name:nyplFullWhite

That being said, we don't do that everywhere.  For example, the `Icon` component does not update the bg color when the icon color is set to white.